smoke-tests-concourse.yml

```diff
@@ -128,3 +128,62 @@ jobs:
       put: notify
       params:
         text_file: properties/failure_message
+
+- name: run-staging-tests
+  plan:
+  - put: metadata
+
+  - task: get-failure-message
+    config:
+      platform: linux
+      image_resource:
+        type: docker-image
+        source:
+          repository: busybox
+      inputs:
+        - name: metadata
+      outputs:
+        - name: properties
+      run:
+        path: sh
+        args:
+          - -exc
+          - |
+            atc_external_url=$(cat metadata/atc_external_url)
+            build_team_name=$(cat metadata/build_team_name)
+            build_pipeline_name=$(cat metadata/build_pipeline_name)
+            build_job_name=$(cat metadata/build_job_name)
+            build_name=$(cat metadata/build_name)
+            echo "Smoke Test Failure: $atc_external_url/teams/$build_team_name/pipelines/$build_pipeline_name/jobs/$build_job_name/builds/$build_name" > properties/failure_message
+
+  - task: build
+    privileged: true
+    config:
+      platform: linux
+      image_resource:
+        type: registry-image
+        source:
+          aws_access_key_id: "((deploy-access-key-id))"
+          aws_secret_access_key: "((deploy-secret-access-key))"
+          aws_region: "((deploy-region))"
+          repository: "govwifi/smoke-tests"
+      run:
+        path: bundle
+        args: ["exec", "rspec"]
+        dir: ../../../usr/src/app
+      inputs:
+      - name: properties
+      params:
+        DOCKER: docker
+        GW_USER: "((GW_USER))"
+        GW_PASS: "((GW_PASS))"
+        GW_2FA_SECRET: "((GW_2FA_SECRET))"
```

Comment:
2FA secret is different for staging (username and password are the same but should probably still be defined separately).